models/demos/t3000/falcon40b/tt/falcon_decoder.py

```diff
@@ -11,7 +11,7 @@
 from models.demos.t3000.falcon40b.tt.falcon_mlp import TtFalconMLP
 from models.utility_functions import torch2tt_tensor, pad_by_zero
 
-from models.demos.t3000.falcon40b.tt.model_utils import convert_to_layout, partial_layernorm
+from models.demos.t3000.falcon40b.tt.model_utils import convert_to_layout, partial_layernorm, fused_partial_layernorm
```

Comment:
partial_layernorm can also be removed here right?

Reply:
right! I removed them here